lib/core/Actions.js

```diff
@@ -283,7 +283,7 @@ async function search(session, client, args = {}) {
   switch (client) {
     case 'YOUTUBE':
       if (args.query) {
-        data.params = Proto.encodeSearchFilter(args.options.period, args.options.duration, args.options.order);
+        data.params = args.options.hasOwnProperty('params') ? args.options.params : Proto.encodeSearchFilter(args.options.period, args.options.duration, args.options.order);
```

Comment:
I think this won't be necessary, I've been working on the protobuf definitions for more search filters.

Reply:
From what I could tell the channels and playlists are only returned if you do not send any params, but maybe I'm missing something?

Reply:
Yup, that's correct.
But I assume that happens because the “type” search filter is missing in the protobuf definition, so YouTube is basically defaulting to “Video” when it should be “All”.

Reply:
@Wykerd 

This should be fixed in commit 43f3c3fbf8edf1793b9352417e81d9ab85fe817a